### DISCUSSION TITLE
How can I get the session in a route handler in the app folder?

### DISCUSSION THREAD

**Discussion by elsueno** [blitz-js/blitz#4293] [Q&A] [answered]
I can use getSession() in an api route in the pages folder. How can I get the session in a route handler in the app folder?

**Comment by siddhsuresh** [comment, marked as answer]
@elsueno you can use `getBlitzContext` to be imported from `blitz-server.ts`, if this is for using a resolver in a `route.ts` file you can directly use `invoke`

documentation of these functions are here: https://blitzjs.com/docs/usage-next-13#blitz-auth (though not specific to a `route.ts` file)
